liquid.py

```diff
@@ -62,6 +63,18 @@ def __init__(self):
             'false',
         )
 
+        self.mount_hoover_path = self.get(
+            'LIQUID_MOUNT_HOOVER_PATH',
```

Comment:
I think `LIQUID_HOOVER_REPOS_PATH` and `LIQUID_LIQUIDINVESTIGATIONS_REPOS_PATH` are more consistent names with the one above.